### ISSUE TITLE
Every time I run updtr it says it's updating nock from 7.2.2 to 7.2.2

### ISSUE THREAD

**Issue by thaggie** [peerigon/updtr#27]
<img width="415" alt="screen shot 2016-03-10 at 10 32 44 am" src="https://cloud.githubusercontent.com/assets/327611/13679900/ffad0660-e6aa-11e5-861d-9e928e70e769.png">

This is what it looks like in package.json:
```
"nock": "^7.2.2",
```

As far as npm is concerned 7.2.2 is the latest version: [https://www.npmjs.com/package/nock](https://www.npmjs.com/package/nock) (as of 2016-03-11)

**Comment by matthaias** [comment]
Hmm, can you show me the output from `npm outdated --json --long --depth=0` in your project folder? Which npm version?

**Comment by thaggie** [comment]
```
{
  "nock": {
    "current": "7.2.2",
    "wanted": "7.4.0",
    "latest": "7.2.2",
    "location": "node_modules/nock",
    "type": "devDependencies"
  }
}
```

**Comment by matthaias** [comment]
Should be solved with `v0.1.8`.
